spec/takeaway_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'takeaway'
+
+RSpec.describe Takeaway do
+
+  subject(:takeaway)  {described_class.new}
+
+    it 'see a list of dishes with prices' do
+      expect(takeaway.display_menu).to eq ('{"chicken"=>3, "rice"=>1, "pizza"=>6, "chips"=>2}')
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/RedundantParentheses: Don't use parentheses around a literal.